packages/react-native-codegen/src/parsers/error-utils.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow strict-local
+ * @format
+ */
+
+'use strict';
+
+import type {ParserType} from './errors';
+
+import {UnusedModuleInterfaceParserError} from './errors';
```

Comment:
Unfortunately, we can't use this syntax.
To import a function, we need to use:
```
const {<function_name>} = require('file_name');
```
See the CircleCI execution for further details! ;)

Reply:
Ohh okay. Sure, I'll make the changes.